src/functions/utils.php

```diff
@@ -885,7 +904,10 @@ function tribe_installed_before( $class, $version ) {
 			return true;
 		}
 
-		return 0 > version_compare( $install_version, $version );
+		return 0 > version_compare(
```

Comment:
I believe this could potentially introduce an issue for users (particularly ones that installed an alpha or beta as their first version). This normalize function removes any awareness of pre-release versions and if there is a change that relies on checking versions to execute code added between a pre-release and a release, then that code would not execute.

Reply:
hm...good point. 
I mean, that was kind of the point - but I was thinking from a QA perspective (there is a work-around). 
Pre-release versions are fairly rare here (although that may change). Suggestion retracted.